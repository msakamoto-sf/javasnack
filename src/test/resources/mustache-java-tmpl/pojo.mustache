booleanValueTrue:{{booleanValueTrue}}
booleanValueFalse:{{booleanValueFalse}}
charValue:{{charValue}}
byteValue:{{byteValue}}
shortValue:{{shortValue}}
intValue:{{intValue}}
longValue:{{longValue}}
floatValue:{{floatValue}}
doubleValue:{{doubleValue}}
fileObject:{{fileObject}}
urlObject:{{urlObject}}
uriObject:{{uriObject}}
inetAddressObject:{{inetAddressObject}}
calendarObject:{{calendarObject}}
dateObject:{{dateObject}}
sqlTimestampObject:{{sqlTimestampObject}}
nullObject:{{nullObject}}
booleanArray:{{#booleanArray}}{{.}},{{/booleanArray}}
charArray:{{#charArray}}{{.}},{{/charArray}}
byteArray:{{#byteArray}}{{.}},{{/byteArray}}
shortArray:{{#shortArray}}{{.}},{{/shortArray}}
intArray:{{#intArray}}{{.}},{{/intArray}}
longArray:{{#longArray}}{{.}},{{/longArray}}
floatArray:{{#floatArray}}{{.}},{{/floatArray}}
doubleArray:{{#doubleArray}}{{.}},{{/doubleArray}}
listObject:(listObject中のarrayが展開されず、IDとなり実行のたびに変化してしまうためテストケースに組み込めず、省略)
mapObject:{{#mapObject}}
str:{{str}}
file:{{file}}
null:{{null}}
byte_array:{{#byte_array}}{{.}},{{/byte_array}}
char_array:{{#char_array}}{{.}},{{/char_array}}
int_array:{{#int_array}}{{.}},{{/int_array}}
list:(listObject中のarrayが展開されず、IDとなり実行のたびに変化してしまうためテストケースに組み込めず、省略)
{{/mapObject}}
日本語プロパティ:{{日本語プロパティ}}
enumObject:{{enumObject}}
enumArray:{{#enumArray}}{{.}},{{/enumArray}}
enumList:{{#enumList}}{{.}},{{/enumList}}
subScope:{{#subScope}}
original:{{original}}
uppercase:{{uppercase}}
{{/subScope}}
subScopes:{{#subScopes}}
original:{{original}}
uppercase:{{uppercase}}
{{/subScopes}}
