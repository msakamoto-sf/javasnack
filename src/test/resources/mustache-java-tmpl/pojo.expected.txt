booleanValueTrue:true
booleanValueFalse:false
charValue:￿
byteValue:127
shortValue:32767
intValue:2147483647
longValue:9223372036854775807
floatValue:3.4028235E38
doubleValue:1.7976931348623157E308
fileObject:
urlObject:http://www.example.com
uriObject:dummy://dummy.dummy/dummy
inetAddressObject:/192.168.1.1
calendarObject:java.util.GregorianCalendar[time&#61;3723456,areFieldsSet&#61;true,areAllFieldsSet&#61;false,lenient&#61;true,zone&#61;sun.util.calendar.ZoneInfo[id&#61;&quot;GMT&quot;,offset&#61;0,dstSavings&#61;0,useDaylight&#61;false,transitions&#61;0,lastRule&#61;null],firstDayOfWeek&#61;1,minimalDaysInFirstWeek&#61;1,ERA&#61;1,YEAR&#61;1970,MONTH&#61;0,WEEK_OF_YEAR&#61;1,WEEK_OF_MONTH&#61;1,DAY_OF_MONTH&#61;1,DAY_OF_YEAR&#61;1,DAY_OF_WEEK&#61;5,DAY_OF_WEEK_IN_MONTH&#61;1,AM_PM&#61;0,HOUR&#61;1,HOUR_OF_DAY&#61;1,MINUTE&#61;2,SECOND&#61;3,MILLISECOND&#61;456,ZONE_OFFSET&#61;0,DST_OFFSET&#61;0]
dateObject:Thu Jan 01 10:02:03 JST 1970
sqlTimestampObject:1970-01-01 10:02:03.456
nullObject:
booleanArray:true,false,
charArray:a,b,c,
byteArray:127,0,-128,
shortArray:32767,0,-32768,
intArray:2147483647,0,-2147483648,
longArray:9223372036854775807,0,-9223372036854775808,
floatArray:3.4028235E38,0.0,1.4E-45,
doubleArray:1.7976931348623157E308,0.0,4.9E-324,
listObject:(listObject中のarrayが展開されず、IDとなり実行のたびに変化してしまうためテストケースに組み込めず、省略)
mapObject:str:HelloJson
file:
null:
byte_array:127,0,-128,
char_array:a,b,c,
int_array:2147483647,0,-2147483648,
list:(listObject中のarrayが展開されず、IDとなり実行のたびに変化してしまうためテストケースに組み込めず、省略)

日本語プロパティ:日本語文字列
enumObject:ONE
enumArray:TWO,FOUR,
enumList:THREE,FIVE,
subScope:original:HelloMustache
uppercase:HELLOMUSTACHE

subScopes:original:subscope1
uppercase:SUBSCOPE1
original:subscope2
uppercase:SUBSCOPE2